app/scripts/controllers/membership.js

```diff
@@ -0,0 +1,304 @@
+'use strict';
+
+angular
+  .module('openshiftConsole')
+  .controller('MembershipController', [
+    '$filter',
+    '$location',
+    '$routeParams',
+    '$scope',
+    '$timeout',
+    '$uibModal',
+    'AuthorizationService',
+    'DataService',
+    'ProjectsService',
+    'MembershipService',
+    'RoleBindingsService',
+    'RolesService',
+    'LocalStorageUserStore',
+    function(
+      $filter,
+      $location,
+      $routeParams,
+      $scope,
+      $timeout,
+      $uibModal,
+      AuthorizationService,
+      DataService,
+      ProjectsService,
+      MembershipService,
+      RoleBindingsService,
+      RolesService,
+      LocalStorageUserStore) {
+
+      var requestContext;
+      var user = LocalStorageUserStore.getUser();
```

Comment:
Are you saying I should do:

```javascript
AuthService
  .withUser()
  then(function(resp) {
    $scope.user = resp; // or resp.<thing>
  });
```
I can update that, didn't know AuthService would give me back the user.